src/Microsoft.Fx.Portability.Offline/Data.cs

```diff
@@ -63,6 +64,20 @@ public static IEnumerable<BreakingChange> LoadBreakingChanges()
             }
         }
 
+        private static IEnumerable<string> GetLocalAllowedCategories()
+        {
+            // Check to see if a file defining valid categories exists
+            var categoriesPath = Path.Combine(GetCurrentDirectoryName(), "BreakingChanges", "BreakingChangeCategories.json");
+            if (File.Exists(categoriesPath))
+            {
+                using (var categoriesFile = File.Open(categoriesPath, FileMode.Open, FileAccess.Read))
```

Comment:
I considered that. It seems like it may be just as useful to be able to run without pre-defined categories, though :-/ I can look into it and maybe add the categories in an embedded file if there's value in that.